### PR TITLE
fixes admin given assassinateonce objective.

### PR DESCRIPTION
**Pull request by Aquilar**
## What Does This PR Do

Fixes https://github.com/ParadiseSS13/Paradise/issues/23174#issue-1978366201

## Why It's Good For The Game

the objective properly functions when it's given to player via admin.

## Testing

spawned two mobs and gave them minds, gave one traitor and a assassinateonce objective, killed the second mob, revived them, added a second assassinate once for comparison and ended the round.

## Changelog
:cl:
fix: Admin given "assassinateonce" works properly now.
/:cl:

### CODE REVIEW

**Comment by lewcc** [comment]
Please fill in the missing parts of the PR template.

File: code/game/gamemodes/objective.dm [hunk 1]
```diff
@@ -57,6 +57,11 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 /datum/objective/proc/found_target()
 	return target
 
+/datum/objective/proc/establish_signals()
```
Comment (Sirryan2002) [inline code/game/gamemodes/objective.dm:60]:
```suggestion
/datum/objective/proc/establish_signals()
	return
```
standard syntax for empty proc

File: code/game/gamemodes/objective.dm [hunk 2]
```diff
@@ -57,6 +57,11 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
...
+	/**
+	 * This is for objectives that need to register signals, so place them in here. Makes it easier for add_objective to call it.
+	 */
```
Comment (Sirryan2002) [inline code/game/gamemodes/objective.dm:63]: move this to before the proc definition

**Review by DGamerL — APPROVED** [review]
LGTM

**Review by S34NW — APPROVED** [review]

**Pull request merged**